packages/ui-library/src/components/forms/textarea/index.css.ts

```diff
@@ -73,7 +76,6 @@ export const { tokenizedLight: textAreaLight, tokenizedDark: textAreaDark } = re
       max-width: 100%;
       word-break: break-all;
       width: 100%;
```

Comment:
Do we need to add the background colour of white here like the ticket says? Something like this maybe? `        background-color: ${SurfaceFill.Default.Hover};`